frontend/src/components/ImplementationReportView/EmbedBadges.tsx

```diff
@@ -1,105 +1,148 @@
-import React, { useState } from "react";
+import React, { useContext, useState } from "react";
 import Container from "react-bootstrap/Container";
-import DropdownButton from "react-bootstrap/DropdownButton";
 import Image from "react-bootstrap/Image";
 import ListGroup from "react-bootstrap/ListGroup";
-import Stack from "react-bootstrap/Stack";
-
+import Button from "react-bootstrap/Button";
+import Modal from "react-bootstrap/Modal";
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
 import CopyToClipboard from "../CopyToClipboard";
 import { Implementation } from "../../data/parseReportData";
 import { Badge, badgesFor } from "../../data/Badge";
+import { Prism as SyntaxHighlighter } from "react-syntax-highlighter";
+import { ThemeContext } from "../../context/ThemeContext";
+import {
+  oneDark,
+  oneLight,
+} from "react-syntax-highlighter/dist/esm/styles/prism";
+import "./EmbedBadges.css";
 
 const EmbedBadges: React.FC<{
   implementation: Implementation;
 }> = ({ implementation }) => {
   const allBadges = badgesFor(implementation);
+  const { isDarkMode } = useContext(ThemeContext);
 
   const [activeFormat, setActiveFormat] = useState(supportedFormats[1]);
   const [activeBadge, setActiveBadge] = useState(allBadges.Metadata[0]);
-
+  const [show, setShow] = useState(false);
   const badgeEmbed = activeFormat.generateEmbed(activeBadge);
 
+  const compareBadges = (a: Badge, b: Badge) => {
```

Comment:
If you're going to define this (which I previously considered but decided it wasn't really any better than just relying on the name alone for now) I think it belongs in `data/Badge`, especially if/when we have a badge class rather than just an interface. But feel free also to leave it as it was before, I don't mind either way personally.

Reply:
I have removed the comparator function and reverted the changes. Thanks.